### PR TITLE
Replace treehouse track with individual rails videos

### PR DESCRIPTION
**Pull request by hackartisan**
The track included all the html, css, javascript, and ruby videos which
we don't do in the same order.

### CODE REVIEW

File: training-curriculum/rails.md [hunk 1]
```diff
@@ -11,9 +11,14 @@
 * Bootstrap 4
 
 ## Videos
-* [Rails development treehouse course](https://teamtreehouse.com/tracks/rails-development)
+* [Rails 5 basis](https://teamtreehouse.com/library/ruby-on-rails-5-basics)
```
Comment (sandbergja) [inline training-curriculum/rails.md:14]:
```suggestion
* [Rails 5 basics](https://teamtreehouse.com/library/ruby-on-rails-5-basics)
```

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks, @hackartisan !  I just saw one small typo.

**Comment by hackartisan** [comment]
Thanks for catching that! I accepted the suggestion.

**Review by sandbergja — APPROVED** [review]
🎉

**Pull request merged**